### PR TITLE
Calendar with link

### PR DESCRIPTION
**Pull request by timmarschall**
Added a link to the secure site that takes you to the calendar. How Handy!

(also fixed an issue that would have brought down the server my b)

### CODE REVIEW

File: SigmaPi/Secure/views.py [hunk 1]
```diff
@@ -18,6 +18,21 @@ def get_url(request):
...
+		key = "ks9c9fb85da3202337"
+	elif (len(groups.filter(name="BAC Chair")) > 0): 
+		key = "ks138f7ef9cf366de5"
+	elif (len(groups.filter(name="House Manager")) > 0): 
+		key = "ks6dddbcfd5ea326f8"
+	elif (len(groups.filter(name="Public Relations Chair")) > 0): 
+		key = "ks148197db7e9a64b8"
+	elif (len(groups.filter(name="Rush Chair")) > 0): 
+		key = "ksfe0960784ea55a4e"
+	elif len(groups.filter(name="Sage")) > 0 or len(groups.filter(name="2nd Counselor")) > 0 or len(groups.filter(name="3rd Counselor")) > 0 or len(groups.filter(name="4th Counselor")) > 0 or len(groups.filter(name="First Counselor")) > 0 or len(groups.filter(name="Herald")) > 0:
```
Comment (Rdbaker) [inline SigmaPi/Secure/views.py]: This seems like a really long way to write this... (hint: https://docs.djangoproject.com/en/1.9/ref/models/querysets/#in)

**Comment by Rdbaker** [comment]
Are we comfortable leaving keys in publicly accessible source code?

File: SigmaPi/Secure/views.py (hunk 1 above)
Comment (timmarschall) [inline SigmaPi/Secure/views.py]: See, I never wrote python before. Ever. Feel free to fix my shitty code.

**Comment by timmarschall** [comment]
@Rdbaker On one hand we shouldn't leave keys in there. But I have no idea how to get theses to be...gone. Maybe we can put them in the db itself?

On the other hand, the keys don't do much except allowing you to edit events. An administrator can always deactivate keys or exchange them if we find fraudulent activity.

Any help is appreciated, I want the calendar to work and be easily accessible to the right people.

**Pull request closed without merging**